src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -71,7 +71,7 @@ public class NodeController : Controller
         private readonly IGetUnspentTransaction getUnspentTransaction;
 
         /// <summary>Specification of the network the node runs on.</summary>
-        private Network network; // Not readonly because of ValidateAddress
+        private readonly Network network; // Not readonly because of ValidateAddress
```

Comment:
Might as well remove the comment from here, I can't see what it was referring to in the first place
```suggestion
        private readonly Network network;
```